docs/source/services/timestamp_service/clustering.rst

```diff
@@ -0,0 +1,30 @@
+========
+Clusters
+========
+
+The Timestamp Service can be run in clustered mode. The main benefit of this is that you get high availability;
+timestamps can be issued as long as a majority of nodes in a cluster are available and can communicate. It may
+be worth noting that there is a performance tradeoff involved here, as distributed consensus requires us to communicate
+with the other servers in the cluster quite frequently (and thus becomes more costly as more servers are added).
+
+The cluster will agree on a leader using leader election. Thereafter, the leader will be the only node that actually
+issues timestamps (other nodes will return a 503). The leader also writes an upper bound for the timestamps that
+it has given out to a persistent store; in practice, we write this to a timestamp table in the key-value store.
+
+Note that the leader still needs to talk to a quorum of nodes before entertaining a timestamp request, to check
```

Comment:
Okay. I should have said one/two round trips _to a quorum of nodes_ - will change this.